obal/data/module_utils/obal.py

```diff
@@ -58,11 +58,11 @@ def get_specfile_nevr(specfile, scl=None, dist=None, macros=None):
     return specfile_macro_lookup(specfile, '%{nevr}', scl=scl, dist=dist, macros=macros)
 
 
-def get_whitelist_status(build_command, tag, package):
+def package_exists(build_command, tag, package):
```

Comment:
I think you need a

```python
if build_command is None:
  build_command = 'koji'
```

for this to work with optional build_command in the modules

Reply:
I'd suggest `if not build_command` to also catch empty strings